packages/cashier/src/stores/__tests__/iframe-store.spec.ts

```diff
@@ -1,13 +1,13 @@
 import IframeStore from '../iframe-store';
 import { configure } from 'mobx';
-import { TRootStore, TWebSocket } from 'Types';
+import { mockStore, TStores } from '@deriv/stores';
```

Comment:
```suggestion
import { mockStore } from '@deriv/stores';
```
@rostislav-deriv Use `TRootStore` instead 🙏🏻